addon/components/new-osf-navbar/template.hbs

```diff
@@ -46,20 +46,28 @@
                             </li>
                         {{else}}
                             {{!SERVICE LINK}}
-                            <li>
-                                <a href="{{navLink.href}}">
-                                    {{#if (eq navLink.type 'addAPreprint')}}
-                                        <span class="hidden-xs hidden-sm">
-                                            {{navLink.name}}
-                                        </span>
-                                        <span class="hidden-md hidden-lg hidden-xl">
-                                            {{t 'eosf.navbar.add'}}
-                                        </span>
-                                    {{else}}
+                            {{#if (eq navLink.name.string "Donate")}}
```

Comment:
I know we do this for the search link, but this will fail for languages other than English. It would be better to do it how addAPreprint is done, adding a type property and checking that.